packages/breadboard-cli/package.json

```diff
@@ -146,5 +146,16 @@
     "serve": "^14.2.1",
     "vite": "^4.4.5",
     "yaml": "^2.3.4"
-  }
+  },
+  "bundleDependencies": [
```

Comment:
I tried to read the docs for "bundleDependencies", and I can't figure out why we need this:
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bundledependencies

Reply:
OMG I just got it. This is because these deps are bundled by vite, and not seen by the deps process otherwise.